jsonformatter_test.go

```diff
@@ -90,9 +91,17 @@ func TestCustomWriter(t *testing.T) {
 				c := JSONFormatter{}
 
 				result, err := c.Format(&logEntry)
+				stringResult := string(result)
 
 				assert.Assert(t, err == nil, "failed custom writer writing: %s", err)
-				assert.Equal(t, string(result), fmt.Sprintf("{\"level\":%d,\"msg\":\"test\",\"time\":%d}\n", testCase.expectedLevel, testCase.expectedTime))
+
+				assert.Equal(t, stringResult, fmt.Sprintf("{\"level\":%d,\"msg\":\"test\",\"time\":%d}\n", testCase.expectedLevel, testCase.expectedTime))
+
+				var timestamp int
+				fmt.Sscanf(stringResult,
+					"{\"level\":"+strconv.Itoa(testCase.expectedLevel)+",\"msg\":\"test\",\"time\":%d}",
+					&timestamp)
+				assert.Assert(t, timestamp >= 1e12 && timestamp <= 1e15, "timestamp is not in milliseconds: %d", timestamp)
```

Comment:
I don't know if it's ok this check.
Here you could check if the year is correct, for example? Or make a convert and assert time.Now() is in the near to the specific log time (parsing the millisecond)?

Reply:
Now I convert the timestamp into date and verify that the year is greater than `2021`